src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -244,7 +246,11 @@ private void OutputBaseSize(ref ObjectDataBuilder objData)
             int minimumObjectSize = pointerSize * 3;
             int objectSize;
 
-            if (_type.IsDefType)
+            if (_type is CanonBaseType)
+            {
+                objectSize = pointerSize;
```

Comment:
As far as a I remember yes, but i will double check. We don't really do anything anyways with the base size for __UniversalCanon (in fact, in NUTC, we return an indeterminate size)

---
In reply to: [97864601](https://github.com/dotnet/corert/pull/2575#discussion_r97864601) [](ancestors = 97864601)